src/main/java/com/speaker/service/validator/message/MessageStatusValidatorImpl.java

```diff
@@ -0,0 +1,35 @@
+package com.speaker.service.validator.message;
+
+import com.speaker.dto.MessageDTO;
+import com.speaker.dto.ValidatorError;
+import com.speaker.entities.EntityField;
+import com.speaker.entities.Status;
+import com.speaker.service.validator.AbstractValidator;
+import com.speaker.service.validator.Validator;
+import com.speaker.service.validator.type.ErrorType;
+import com.speaker.service.validator.type.FieldType;
+import org.springframework.stereotype.Component;
+
+import java.util.Objects;
+
+@Component
+public class MessageStatusValidatorImpl extends AbstractValidator implements Validator<MessageDTO> {
+
+    @Override
+    public ValidatorError validate(EntityField entityField) {
+        if (!Objects.equals(entityField.getField().toString(), Status.NEW.name())) {
```

Comment:
you get NPE
just `Status.NEW.equals(entityField.getField())`

Reply:
Comments accepted, corrections made.